portal/views/portal.py

```diff
@@ -845,6 +855,14 @@ def settings():
             locale_code=user.locale_code)
 
     if form.patient_id.data and form.timestamp.data:
+        # TODO: is this in use?  would change all a user's QNR authored
+        # dates - guessing it is dead.  Remove soon unless needed
+        msg = (
```

Comment:
Is this the form to back-date QuestionnaireResponses? If so, might still be helpful for testing

Reply:
that was the original intended use - @mcjustin used it for exactly that.  since then we've added the ability to easily change a users consent date.